typescript/sdk/src/gas/token-prices.ts

```diff
@@ -84,6 +84,11 @@ export class CoinGeckoTokenPriceGetter implements TokenPriceGetter {
     this.metadata = chainMetadata;
   }
 
+  static new(expirySeconds?: number): CoinGeckoTokenPriceGetter {
```

Comment:
nit: `new` is a weird name for a static factory
maybe `from`?

Reply:
agree this is weird - changed to `withDefaultCoinGecko`